data/transactions/logic/doc.go

```diff
@@ -316,6 +316,7 @@ var txnFieldDocs = map[string]string{
 	"AssetCloseTo":             "32 byte address",
 	"GroupIndex":               "Position of this transaction within an atomic transaction group. A stand-alone transaction is implicitly element 0 in a group of 1",
 	"TxID":                     "The computed ID for this transaction. 32 bytes.",
+	"CreatableID":              "The assigned creatable ID for an asset or application creation transaction, or 0 if the transaction does not create anything.",
```

Comment:
this is a pseudo field, seems like the first one. Need to mention that and the fact it is apps-only.
Maybe a new opcode? :)

Reply:
GroupIndex is kind of a pseudo field, as is TxID.  (It's a hash that is not actually included in the txn.)

Reply:
We've concluded @algorandskiy is right.  Although there may be other pseudo fields, this would have been the first that depended on state unknown until the txn is being put on chain.  That would have made it a stateful only field.  That's weird.  We're going with `gaid` group asset/app id. Which will take a slot, like the other g-opcodes.

Reply:
It may be that we want to clean this up by AVM 1.0, so that we have a new `txn`-like opcode that is stateful only - it's for inspecting transaction specific things that are computed are proposal time.  It would subsume this opcode, the gload opcodes, and perhaps provide access to previous txns' apply data.